var/spack/repos/builtin/packages/shuffile/package.py

```diff
@@ -10,16 +10,17 @@ class Shuffile(CMakePackage):
     """Shuffle files between MPI ranks"""
 
     homepage = "https://github.com/ecp-veloc/shuffile"
-    url      = "https://github.com/ecp-veloc/shuffile/archive/v0.0.3.zip"
+    url      = "https://github.com/ecp-veloc/shuffile/archive/v0.0.4.tar.gz"
     git      = "https://github.com/ecp-veloc/shuffile.git"
 
     tags = ['ecp']
 
     version('master', branch='master')
+    version('0.0.4', sha256='ceeedd1bb1620e0a21aa99c2b766ceaf060a45e1b4e5037c6ced61d8ef8139b0')
     version('0.0.3', sha256='6debdd9d6e6f1c4ec31015d7956e8b556acd61ce31f757e4d1fa5002029c75e2')
```

Comment:
Same issue as https://github.com/spack/spack/pull/22977#discussion_r613605901

Reply:
Updated the hash.